From 07ab6bda0746d998b25d9f3ea82afb06d004b903 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Wed, 7 Jun 2017 18:23:38 -0400
Subject: [PATCH] autoboot

---
 include/configs/am335x_evm.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index c31b373..0d1ce84 100644
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -29,6 +29,9 @@
 
 #define CONFIG_SYS_BOOTM_LEN		(16 << 20)
 
+#undef CONFIG_BOOTDELAY
+#define CONFIG_BOOTDELAY        (-2) /* Always autoboot */
+
 #define MACH_TYPE_TIAM335EVM		3589	/* Until the next sync */
 #define CONFIG_MACH_TYPE		MACH_TYPE_TIAM335EVM
 #define CONFIG_BOARD_LATE_INIT
-- 
2.7.4

