From 0d43846d50636195e37001c220be9ac67504dd25 Mon Sep 17 00:00:00 2001
From: Frank Hunleth <fhunleth@troodon-software.com>
Date: Wed, 7 Jun 2017 18:23:38 -0400
Subject: [PATCH] autoboot

---
 include/configs/am335x_evm.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/configs/am335x_evm.h b/include/configs/am335x_evm.h
index e49a487..64b6920 100644
--- a/include/configs/am335x_evm.h
+++ b/include/configs/am335x_evm.h
@@ -22,6 +22,9 @@
 # define CONFIG_TIMESTAMP
 #endif
 
+#undef CONFIG_BOOTDELAY
+#define CONFIG_BOOTDELAY                (-2) /* Always autoboot */
+
 #define CONFIG_SYS_BOOTM_LEN		(16 << 20)
 
 #define CONFIG_MACH_TYPE		MACH_TYPE_AM335XEVM
-- 
2.7.4

